README.rst

```diff
@@ -449,9 +492,17 @@ The following example shows the different configuration options available in
 the run step:
 
 .. code:: yaml
-
+  
+  version: 2.0
+  # Optional map to specify the buildrunner configuration syntax version
```

Comment:
I feel like this comment should go above the `version` config parameter. And probably say `Optional buildrunner configuration syntax version`